manifests/k8s-spot-termination-handler/k8s-spot-termination-handler.yaml

```diff
@@ -66,7 +66,7 @@ spec:
       serviceAccountName: k8s-spot-termination-handler
```

Comment:
is SA here for future reference only , Normally we use SA with role binding for the specific deployment to manage pod privileges inside k8s but cant see and role or role binding